examples/docs_snippets/docs_snippets/concepts/logging/python_logger.py

```diff
@@ -23,10 +23,10 @@ def ambitious_op():
 
     # end_python_logger
     @graph
-    def thing():
+    def thing_one():
```

Comment:
any reason for this change?

Reply:
Adding two graphs with the same name to the same repository previously did not raise an error before this PR, even though it should because the graphs will be coerced to jobs. This was an edge case I caught and fixed in this PR, and this test had to be changed in order to not error afterwards.

Reply:
ohhhh good point. that's great that we catch this now. 👍